src/System.Management.Automation/FormatAndOutput/common/ListWriter.cs

```diff
@@ -86,12 +87,26 @@ internal void Initialize(string[] propertyNames, int screenColumnWidth, DisplayC
                 if (propertyNameCellCounts[k] < _propertyLabelsDisplayLength)
                 {
                     // shorter than the max, add padding
-                    _propertyLabels[k] = propertyNames[k] + StringUtil.Padding(_propertyLabelsDisplayLength - propertyNameCellCounts[k]);
+                    if (ExperimentalFeature.IsEnabled("PSAnsiRendering"))
+                    {
+                        _propertyLabels[k] = PSStyle.Instance.Formatting.FormatAccent + propertyNames[k] + StringUtil.Padding(_propertyLabelsDisplayLength - propertyNameCellCounts[k]);
+                    }
+                    else
+                    {
+                        _propertyLabels[k] = propertyNames[k] + StringUtil.Padding(_propertyLabelsDisplayLength - propertyNameCellCounts[k]);
+                    }
                 }
                 else if (propertyNameCellCounts[k] > _propertyLabelsDisplayLength)
                 {
                     // longer than the max, clip
-                    _propertyLabels[k] = propertyNames[k].Substring(0, dc.GetHeadSplitLength(propertyNames[k], _propertyLabelsDisplayLength));
+                    if (ExperimentalFeature.IsEnabled("PSAnsiRendering"))
+                    {
+                        _propertyLabels[k] = PSStyle.Instance.Formatting.FormatAccent + propertyNames[k].Substring(0, dc.GetHeadSplitLength(propertyNames[k], _propertyLabelsDisplayLength));
+                    }
+                    else
+                    {
+                        _propertyLabels[k] = propertyNames[k].Substring(0, dc.GetHeadSplitLength(propertyNames[k], _propertyLabelsDisplayLength));
+                    }
```

Comment:
shouldn't this apply to all of these (it seems that the case where propertyNameCellCounts[k] == _propertyLabelsDisplayLength is missing)? Would it be better to have the logic for the Experimental feature be before the `+Separator`? something like:
```
  if (ExperimentalFeature.IsEnabled("PSAnsiRendering"))
  {
 _propertyLabels[k] = PSStyle.Instance.Formatting.FormatAccent +  _propertyLabels[k];
}

 _propertyLabels[k] += Separator;
```

Reply:
So the way this existing code works is if the label is shorter than the calculated width, it pads it.  If longer, it truncates.  If it's `=`, then it falls into the else statement below and is used without transformation.

Reply:
so that means you're not coloring those? is that what you want?

Reply:
I misunderstood your comment and now understand that the else clause also needs to be wrapped by the experimental feature.